examples/examples/storage_iterating.rs

```diff
@@ -52,14 +52,18 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         // Fetch at most 10 keys from below the prefix XcmPallet' VersionNotifiers.
         let keys = api
             .storage()
-            .fetch_keys(&key_addr.to_root_bytes(), 10, None, None)
+            .at(None)
+            .await?
+            .fetch_keys(&key_addr.to_root_bytes(), 10, None)
```

Comment:
IMHO, I like this new API better it's easier to read with `fn at` :+1:

Reply:
Thanks for the review!